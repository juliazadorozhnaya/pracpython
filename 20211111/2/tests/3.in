Hello my friend
how are you 
(0,0), (1,-1), (2,3)
