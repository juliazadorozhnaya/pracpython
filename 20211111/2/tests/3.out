Invalid input
Invalid input
2.50
